package.json

```diff
@@ -6,10 +6,11 @@
     "url": "https://github.com/apiaryio/Paw-APIBlueprintImporter"
   },
   "scripts": {
-    "test": "mocha",
+    "archive": "cake archive",
     "build": "cake build",
     "install": "cake install",
-    "archive": "cake archive",
+    "lint": "eslint src/APIElementImporter.js test/",
```

Comment:
The main entrypoint `src/APIBlueprintImporter.js` cannot confirm as it contains JS syntax that isn't in the ES6 standard used in Paw.